packages/lesswrong/components/posts/PostsNewForm.tsx

```diff
@@ -273,19 +282,27 @@ const PostsNewForm = () => {
           ? sanitizeEditableFieldValues(prefilledProps, ['contents'])
           : prefilledProps;
 
-        const moderationGuidelinesField = currentUserWithModerationGuidelines.moderationGuidelines?.originalContents && hasAuthorModeration()
+
+        const hasValidModerationGuidelines =
+          currentUserWithModerationGuidelines.moderationGuidelines?.originalContents?.data !== undefined &&
```

Comment:
Shoggoth added this of it's own initiative. I... weakly think we can remove it but wanted to talk through it.